test/expectations/selection_ranges/begin_rescue_ensure.exp.json

```diff
@@ -14,52 +14,40 @@
                 },
                 "end": {
                     "line": 5,
-                    "character": 6
+                    "character": 17
                 }
             },
             "parent": {
                 "range": {
                     "start": {
-                        "line": 5,
-                        "character": 2
+                        "line": 4,
+                        "character": 0
                     },
                     "end": {
                         "line": 5,
-                        "character": 18
+                        "character": 17
                     }
                 },
                 "parent": {
                     "range": {
                         "start": {
-                            "line": 4,
+                            "line": 2,
                             "character": 0
                         },
                         "end": {
-                            "line": 6,
-                            "character": 0
+                            "line": 5,
+                            "character": 17
                         }
                     },
                     "parent": {
                         "range": {
                             "start": {
-                                "line": 2,
+                                "line": 0,
                                 "character": 0
                             },
                             "end": {
-                                "line": 6,
-                                "character": 0
-                            }
-                        },
-                        "parent": {
```

Comment:
If it helps for reviewing, I can add a note to explain each 'substantial' change in the expectations.